packages/adblocker/src/engine/engine.ts

```diff
@@ -350,6 +476,7 @@ export default class FilterEngine extends EventEmitter<EngineEventHandlers> {
     preprocessors?: Preprocessor[];
     lists?: Map<string, string>;
     config?: Partial<Config>;
+    engineVersion?: number;
```

Comment:
Should this be reverted?

Reply:
Thanks for pointing, fixed in [`52a07ac` (#4138)](https://github.com/ghostery/adblocker/pull/4138/commits/52a07ac5670f50d05dd602dff7e56ced9f772170)